src/wizard/react-native/index.md

```diff
@@ -5,63 +5,38 @@ support_level: production
 type: language
 ---
 
-## Integrating the SDK
+Sentry captures data by using an SDK within your application’s runtime.
 
-Sentry captures data by using an SDK within your application’s runtime. These are platform-specific and allow Sentry to have a deep understanding of how your application works.
+> If you are using Expo, see [How to Add Sentry to Your Expo Project](https://docs.expo.io/guides/using-sentry/). This SDK works for both managed and bare projects.
 
-> If you are using `expo-cli` you need to use another SDK see: [https://docs.expo.io/versions/latest/guides/using-sentry/](https://docs.expo.io/versions/latest/guides/using-sentry/)
-> This SDK only works for ejected projects or projects that directly use React Native.
 
-
-To install the React Native SDK, add the dependency to your project by either using `npm` or `yarn`.
+Install our React Native SDK using either `yarn` or `npm`:
 
 ```bash
-npm install @sentry/react-native --save
+npm install --save @sentry/react-native
 # or
 yarn add @sentry/react-native
 ```
 
 ### Linking
 
-Since our SDK also supports native crashes, we need to link the SDK to your native projects.
-
-Above `react-native >= 0.60` you need to do:
+Link the SDK to your native projects to enable native crash reporting. If you are running a project with `react-native < 0.60`, you still need to call `react-native link`, otherwise you can skip this step as `react-native >=0.60` does this automatically.
 
 ```bash
-npx sentry-wizard -i reactNative -p ios android
+npx @sentry-wizard -i reactNative -p ios android
```

Comment:
Heads up @PeloWriter this would not work as `@sentry-wizard` doesn't exist on npm, it would be `@sentry/wizard`